src/questions/questions.model.ts

```diff
@@ -1,39 +1,87 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Quiz } from '../quiz/quiz.model';
+import { Quiz_Question } from '../quiz/quiz.question.model';
 
-@Table({ tableName: 'questions', createdAt: false, updatedAt: false})
+@Table({ tableName: 'questions', createdAt: false, updatedAt: false })
 export class Question extends Model<Question, QuestionCreationAttrs> {
-
-  @ApiProperty({example: '1', description: 'uniq id'})
-  @Column({ type: DataType.INTEGER, unique: true, autoIncrement: true, primaryKey: true })
+  @ApiProperty({ example: '1', description: 'uniq id' })
+  @Column({
+    type: DataType.INTEGER,
+    unique: true,
+    autoIncrement: true,
+    primaryKey: true,
+  })
   id: number;
 
-  @ApiProperty({example: 'NodeJS question #13', description: 'required title / not uniq / string'})
+  @ApiProperty({
+    example: 'NodeJS question #13',
+    description: 'required title / not uniq / string',
+  })
   @Column({ type: DataType.STRING, allowNull: false })
   title: string;
 
-  @ApiProperty({example: '{"question": "blablabla", "answers": { "wrong" : "bla", "correct" : "blabla"}}', description: 'sample of question at JSON datatype/ required'})
+  @ApiProperty({
+    example:
+      '{"question": "blablabla", "answers": { "wrong" : "bla", "correct" : "blabla"}}',
+    description: 'sample of question at JSON datatype/ required',
+  })
   @Column({ type: DataType.JSON, allowNull: false })
   content: JSON;
 
-  @ApiProperty({example: 'oneChoice / multiChoice', description: 'how many of variables can been chosen for answer / required'})
-  @Column({ type: DataType.ENUM({ values: Object.values(QuestionTypes) }), allowNull: false })
+  @ApiProperty({
+    example: 'oneChoice / multiChoice',
+    description: 'how many of variables can been chosen for answer / required',
+  })
+  @Column({
+    type: DataType.ENUM({ values: Object.values(QuestionTypes) }),
+    allowNull: false,
+  })
   type: QuestionTypes;
 
-  @ApiProperty({example: 'easy / normal / hard', description: 'set your difficulty / required'})
-  @Column({ type: DataType.ENUM({ values: Object.values(DifficultyTypes) }), allowNull: false })
+  @ApiProperty({
+    example: 'easy / normal / hard',
+    description: 'set your difficulty / required',
+  })
+  @Column({
+    type: DataType.ENUM({ values: Object.values(DifficultyTypes) }),
+    allowNull: false,
+  })
   difficulty: DifficultyTypes;
 
-  @ApiProperty({example: 'easy / normal / hard', description: 'set your difficulty / required'})
+  @ApiProperty({
+    example: 'that test will show us ur power in Node js',
+    description: 'make it by ur own / required',
+  })
   @Column({ type: DataType.STRING, allowNull: false })
   description: string;
 
-  @ApiProperty({example: '1', description: 'number of topic that should be import into question'})
+  @ApiProperty({
+    example: 'that test will show us ur power in Node js',
+    description: 'make it by ur own / required',
```

Comment:
Copy error. Replace with a record from create-question.dto.ts: "timer in seconds / required'"